crates/bevy_ecs/src/component/mod.rs

```diff
@@ -13,7 +14,7 @@ use thiserror::Error;
 /// A component is data associated with an [`Entity`](crate::entity::Entity). Each entity can have
 /// multiple different types of components, but only one of them per type.
 ///
-/// Any type that is `Send + Sync + 'static` automatically implements `Component`.
+/// Any type that is `Send + Sync + 'static` can implement `Component` using `#[derive(Component)]`.
```

Comment:
I would like a comment here describing how to work around the orphan rules for new Rust users.